app/screens/onboarding/footer_buttons.tsx

```diff
@@ -32,6 +35,8 @@ const getStyleSheet = makeStyleSheetFromTheme((theme: Theme) => ({
     nextButtonText: {
         flexDirection: 'row',
         position: 'absolute',
+        justifyContent: 'center',
+        minWidth: 100,
```

Comment:
Why `minWidth` is used to solve the issue? Shouldn't the button grow according to the content? 0/5
(Similar for the other minWidth in this file)

Reply:
what about using `flexGrow` instead?

Reply:
I removed the minWidth and use the button fixed width.

>Shouldn't the button grow according to the content?

it should, but since we are using an animation with interpolation for the width (the animation between the last and the previous slide), then that is causing it to have a fixed width from the initial render. flexGrow didn't work neither. So, by setting the width to 120 works well for all the language translations for this word. Actually Spanish is the longest with 9 letters.